torch/_ops.py

```diff
@@ -54,6 +54,9 @@ def __init__(self, name):
         self.name = name
 
     def __getattr__(self, op_name):
+        # It is not a valid op_name when __file__ is passed in
+        if op_name == '__file__':
+            return 'torch.ops'
```

Comment:
maybe torch._ops?

Reply:
Discussed offline and decided to go with 'torch.ops' since the external module name always appears as torch.ops.*.